### PR TITLE
[IMP] move china city data into separate module `i10n_cn_city`.

### PR DESCRIPTION
**Pull request by jeffery9**
Description of the issue/feature this PR addresses:

improvement of #23879 

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_cn_city/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline addons/l10n_cn_city/__init__.py]: coding line is not necessary in python 3

File: addons/l10n_cn_city/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# Copyright (C) 2018-now  jeffery9@gmail.com
+
+{
+    'name': 'China - City Data',
+    'version': '1.8',
+    'category': 'Localization',
+    'author': 'Jeffery Chen Fan<jeffery9@gmail.com>',
+    'description': """
+Include those data/包含以下数据
```
Comment (elicoidal) [inline addons/l10n_cn_city/__manifest__.py]: "Includes the following data for the Chinese localization"

File: addons/l10n_cn_city/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
```
Comment (elicoidal) [inline addons/l10n_cn_city/__manifest__.py]: remove empty line

File: addons/l10n_cn_city/__init__.py (hunk 1 above)
Comment (mreficent) [reply addons/l10n_cn_city/__init__.py]: odoo still tries to be compatible with python 2

**Comment by mart-e** [comment]
Merged at c3f0915b2b49d100257a1ac0f20d56beb3307d0d, thanks

**Pull request closed without merging**